index.html

```diff
@@ -31,15 +48,15 @@ <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
                 <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box1">Box 2</div>
```

Comment:
Great job here, Vanesa! Let me show you another way of handing this problem so you can avoid doing extra work. 

The initial settings for the HTML looked like this:

``` <div class='box'>box1</div> ```   (every box was set like this)

Another way of tackling this problem is to simply add an additional class -

<div class='box box1'>box1</div> 

This will free up our CSS because instead of having to add the box dimensions, along with the color, we can simply just list the color with our new class.

``` .box1{
      background: teal; 
}
```

Your answer isn't wrong. This is just a way to avoid the repetition of having to add the dimensions of the box with each new class that you create. Something to consider for future projects!